library/build.gradle

```diff
@@ -37,6 +37,10 @@ android {
     lintOptions {
         abortOnError false
     }
+
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
:x: this option should never be turned on.  It means you're testing things not under our control.

Can discuss later :x: